### DISCUSSION TITLE
Help migrating users to Microsoft Entra Id authentication.

### DISCUSSION THREAD

**Discussion by mike-credisense** [datalust/seq-tickets#2154] [Help] [answered]
I'm looking into using Microsoft Entra Id for authentication however I need to migrate 3 existing username and password user accounts. 

What do I need to do for this?

**Comment by nblumhardt** [comment, marked as answer]
Hi Mike, thanks for getting in touch.

Before changing the authentication provider in Settings > System, you'll need to change each existing user's _username_ (not email address) to their username in Entra ID, which is most likely an email address.

After this you should be able to enable Entra ID and have everything line up automatically.

Hope this helps,
Nick

**Reply by mike-credisense** [reply]
Thanks 
Seems to be working but one user lost their dashboard

**Reply by nblumhardt** [reply]
Thanks for the follow-up.

That's strange; is it possible that they need to search for it in the dashboard list? Or another possibility - is there now a duplicate account for that user under Settings > Users?

The authentication provider shouldn't have any impact on other stored data, it's really only used for login/session establishment :thinking:
